unstructured_ingest/v2/processes/connectors/delta_table.py

```diff
@@ -94,14 +94,17 @@ class DeltaTableUploader(Uploader):
     connection_config: DeltaTableConnectionConfig
     connector_type: str = CONNECTOR_TYPE
 
-    @requires_dependencies(["s3fs", "fsspec"], extras="s3")
+    @requires_dependencies(["s3fs", "fsspec", "boto3"], extras="s3")
```

Comment:
This should not require the s3 extra for the delta table connector, also can we simply do all the checks with boto3 then? Rather than needing the fsspec dependency? 
```suggestion
    @requires_dependencies(["boto3"], extras="delta-table")
```

Reply:
Addressed suggestion in the commit